public/apps/account/account-nav-button.tsx

```diff
@@ -186,3 +186,23 @@ export function AccountNavButton(props: {
     </EuiHeaderSectionItemButton>
   );
 }
+
+export function reloadAfterTenantSwitch(): void {
+  // the below portion is to clear URLs starting with 'lastUrl'
+  // when switching tenants, the last URLs will be from the old tenancy therefore we need to remove these from sessionStorage.
+  const lastUrls = [];
+
+  for (let i = 0; i < window.sessionStorage.length; i++) {
```

Comment:
could you declare `const { sessionStorage } = window;` and then use it as sessionStorage across the file? This will help with readability too

Reply:
complete, thanks @DarshitChanpura